content/en/docs/setup/independent/kubelet-integration.md

```diff
@@ -0,0 +1,142 @@
+to make the kubelet pick up the latest dynamic flags which were previously written. After that the rest of the regular
+`kubeadm init` workflow continues.
+
+On the other hand, `kubeadm join` uses the Bootstrap Token credential to download the `kubelet-config-1.X` ConfigMap
+and write it to `/var/lib/kubelet/config.yaml`.
+
+The dynamic environment file is generated in exactly the same way as `kubeadm init`.
+
+After these files are written, `kubeadm` attempts a `systemctl daemon-reload && systemctl restart kubelet` in order
+to make the kubelet pick up the latest dynamic flags written.
+
+Then `kubeadm join` performs the discovery, which yields the `/etc/kubernetes/bootstrap-kubelet.conf` KubeConfig
+file consisting of a CA certificate and Bootstrap Token.
+
+At that point the Bootstrap Token is used by the kubelet to perform the TLS Bootstrap and to get the unique credential which
+is stored in `/etc/kubernetes/kubelet.conf`. As of kubeadm v1.11, `kubeadm join` waits for the `/etc/kubernetes/kubelet.conf`
+file to appear on disk, which means that the kubelet has performed the TLS Bootstrap.
+
+###  The kubelet drop-in file that kubeadm uses
+Here are the contents of the the drop-in file the kubeadm _deb/rpm package_ ships for the kubelet. It is installed in
+the path `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`.
+
+```
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf
+--kubeconfig=/etc/kubernetes/kubelet.conf"
+Environment="KUBELET_CONFIG_ARGS=--config=/var/lib/kubelet/config.yaml"
+# This is a file that "kubeadm init" and "kubeadm join" generates at runtime, populating
+the KUBELET_KUBEADM_ARGS variable dynamically
+EnvironmentFile=-/var/lib/kubelet/kubeadm-flags.env
+# This is a file that the user can use for overrides of the kubelet args as a last resort. Preferably,
+#the user should use the .NodeRegistration.KubeletExtraArgs object in the configuration files instead.
+# KUBELET_EXTRA_ARGS should be sourced from this file.
+EnvironmentFile=-/etc/default/kubelet
+ExecStart=
+ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_KUBEADM_ARGS $KUBELET_EXTRA_ARGS
+```
+
+
+Notes:
+- The KubeConfig file to use for the TLS Bootstrap is specified as `/etc/kubernetes/bootstrap-kubelet.conf`,
+but it is not used if `/etc/kubernetes/kubelet.conf` already exists.
+- The KubeConfig file with the unique kubelet identity is `/etc/kubernetes/kubelet.conf`.
+- The file containing the kubelet's ComponentConfig is `/var/lib/kubelet/config.yaml`.
+- The dynamic environment file that contains `KUBELET_KUBEADM_ARGS` is sourced from `/var/lib/kubelet/kubeadm-flags.env`.
+- The file that can contain user specified flag overrides with `KUBELET_EXTRA_ARGS` is sourced from `/etc/default/kubelet`
+(for debs), or `/etc/systconfig/kubelet` (for rpms). Note that `KUBELET_EXTRA_ARGS` is last in the flag chain,
+therefore it has the highest priority.
+
```

Comment:
Here some wrong  format output, may be add a blank line below `Notes` make it work.
https://deploy-preview-9652--kubernetes-io-master-staging.netlify.com/docs/setup/independent/kubelet-integration/